### DISCUSSION TITLE
bigDecimal.add is not a function(there is a "default" field)

### DISCUSSION THREAD

**Discussion by SilentFlute** [royNiladri/js-big-decimal#114] [Q&A]
this pkg is great, and light weight, i like it, but when i trying to use it, i got some troubles...

i add `js-big-decimal` with this cmd:
`$ yarn add js-big-decimal`
and create a `js` file: `foo.js`, and typed these codes in it:
```
const bigDecimal = require('js-big-decimal');

console.log(bigDecimal.add('12', '45'));
```
and then i did this:
`$ node foo.js`
as least, i got this:
`bigDecimal.add is not a function`

then i did some debug things, i figured out: there is a 'default' filed, so i changed the code like this:
```
const bigDecimal = require('js-big-decimal');

console.log(bigDecimal.default.add('12', '45'));
```
then it went will

but the doc didn't mention that, am i did a wrong way or sth? and what's the 'default' field used for?

**Comment by royNiladri** [comment]
This should not be happening. Looks like some bundling. Will have to take a look. For the time being, you can use 1.4 based versions. Those will not have this issue.
